[16.0][ADD] hr_timesheet_overtime_rate_holidays (#34)
coopiteasy/cie-timesheet: hr_timesheet_overtime_rate_holidays/readme/DESCRIPTION.rst

> When timesheet lines are made from a holiday, they are never counted as overtime.

Reviewer: ```suggestion
Ensures that no overtime rate is applied when timesheet lines are created from a holiday.
```
